api/camunda/camunda-openapi.yaml

```diff
@@ -894,7 +1267,6 @@ components:
             type: string
         priority:
           type: integer
-          description: The priority of a user task. The higher the value the higher the priority.
```

Comment:
change-at-the-source, regression, me:

Description needs to be backported from #4178.

Reply:
https://github.com/camunda/camunda/pull/22037